pkgs/games/xash3d/default.nix

```diff
@@ -0,0 +1,80 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3
+, pkg-config
+, wafHook
+, SDL2
+, libopus
+, freetype
+, fontconfig
+, dedicated ? false # dedicated server
+, gamedir ? "valve"
+, enableBsp2 ? false # bsp2 support (for quake)
+, enableGles1 ? false # gles1 renderer (nanogl)
+, enableGles2 ? false # gles2 renderer (glwes)
+, enableGl4es ? false # gles2 renderer (gl4es)
+, enableGl ? true # opengl renderer
+, enableSoft ? true # soft renderer
+, enableUtils ? false # mdldec
+}:
+
+stdenv.mkDerivation {
+  pname = "xash3d-unwrapped";
+  version = "unstable-2023-07-02";
+
+  src = fetchFromGitHub {
+    owner = "FWGS";
+    repo = "xash3d-fwgs";
+    rev = "5e878aae89ab1517eeeb259191bd6f2f88a6eccb";
+    fetchSubmodules = true;
+    sha256 = "IrKWF7TYL9QV5UCHjm1rxZ4ZGT+Hmoq0nOj+X+GoiPQ=";
+  };
+
+  patches = [
+    # waf fails to build otherwise
+    ./change-zip-date.patch
+  ];
+
+  nativeBuildInputs = [
+    python3
+    pkg-config
+    wafHook
+  ];
+
+  buildInputs = [
+    SDL2
+    libopus
+    freetype
+    fontconfig
+  ];
+
+  wafConfigureFlags = [ "-P" "-T release" ]
```

Comment:
Nitpick, could you use `--enable-packaging` and `--build-type=release` instead? It's easier to see at a glance what it's doing.

Reply:
I suppose. I've always preferred using short flags, but I did as you asked